src/gc/handletablecore.cpp

```diff
@@ -510,8 +510,8 @@ BOOL SegmentInitialize(TableSegment *pSegment, HandleTable *pTable)
     // we want to commit enough for the header PLUS some handles
     uint32_t dwCommit = HANDLE_HEADER_SIZE;
 
-    // Round down to the dwPageSize
-    dwCommit &= ~(OS_PAGE_SIZE - 1);
+    // Round up to the OS_PAGE_SIZE
+    dwCommit = (dwCommit + OS_PAGE_SIZE - 1) & ~(OS_PAGE_SIZE - 1);
```

Comment:
You can use `ALIGN_UP`:  `size_t dwCommit = ALIGN_UP(HANDLE_HEADER_SIZE, OS_PAGE_SIZE);`

Reply:
Hmm, I have thought we have that macro in PAL only. Obviously I was wrong.